### DISCUSSION TITLE
Recommended budget GPU for LLM experimentation

### DISCUSSION THREAD

**Discussion by msfasha** [ggml-org/llama.cpp#6418] [Q&A]
I want to get a budget GPU for LLM training/fine tuning/experimentation.
I am considering:

a single rtx 4070 ti super 16gb
or a single rtx 4080 16gb
or some dual lower level gpus setting, e.g. dual rtx 4070 12gb ram

Any thoughts, advices or other recommendations?

**Comment by sorasoras** [comment]
For finetune, I think 24G isminimum so a 3090
